dev/create-release/release-build.sh

```diff
@@ -290,6 +290,8 @@ if [[ "$1" == "docs" ]]; then
   cd docs
   # TODO: Make configurable to add this: PRODUCTION=1
   PRODUCTION=1 RELEASE_VERSION="$SPARK_VERSION" jekyll build
+  ret=$?
+  if [[ $ret != 0 ]]; then exit $ret; fi
```

Comment:
Wait .. `set -e` is in effect. Just to be clear, `jekyll build` exits with non-zero code but `set -e` fails to catch it? I am fine as is if it fixes the issue anyway.

Reply:
oh very excellent point. I tested in a script without `set -e`, and did see `set -e` wondering a bit what it wasn't doing.

I guess more digging through the jekyll code.

I did find the `wait` for make_binary_release won't work through - it was said to return exit code 0 if the `wait` was without process ID.